src/test/java/hobbylist/logic/commands/FindCommandTest.java

```diff
@@ -14,59 +13,75 @@
 import hobbylist.testutil.TypicalActivities;
 
 /**
  * Contains integration tests (interaction with the Model) for {@code FindCommand}.
  */
 public class FindCommandTest {
-    private Model model = new ModelManager(TypicalActivities.getTypicalHobbyList(), new UserPrefs());
-    private Model expectedModel = new ModelManager(TypicalActivities.getTypicalHobbyList(), new UserPrefs());
+    private final Model model = new ModelManager(TypicalActivities.getTypicalHobbyList(), new UserPrefs());
+    private final Model expectedModel = new ModelManager(TypicalActivities.getTypicalHobbyList(), new UserPrefs());
 
     @Test
     public void equals() {
-        NameOrDescContainsKeywordsPredicate firstPredicate =
+        NameOrDescContainsKeywordsPredicate firstKeywordPredicate =
                 new NameOrDescContainsKeywordsPredicate(Collections.singletonList("first"));
-        NameOrDescContainsKeywordsPredicate secondPredicate =
+        DateMatchesGivenDatePredicate firstDatePredicate = new DateMatchesGivenDatePredicate("2022-01-01");
+        RatingMatchesGivenValuePredicate firstRatingPredicate = new RatingMatchesGivenValuePredicate(3);
+        NameOrDescContainsKeywordsPredicate secondKeywordPredicate =
                 new NameOrDescContainsKeywordsPredicate(Collections.singletonList("second"));
+        DateMatchesGivenDatePredicate secondDatePredicate = new DateMatchesGivenDatePredicate("2021-02-02");
+        RatingMatchesGivenValuePredicate secondRatingPredicate = new RatingMatchesGivenValuePredicate(2);
 
-        FindCommand findFirstCommand = new FindCommand(firstPredicate);
-        FindCommand findSecondCommand = new FindCommand(secondPredicate);
+        FindCommand findFirstCommand = new FindCommand(firstKeywordPredicate,
+                                                        firstDatePredicate,
+                                                        firstRatingPredicate);
+        FindCommand findSecondCommand = new FindCommand(secondKeywordPredicate,
+                                                        secondDatePredicate,
+                                                        secondRatingPredicate);
 
         // same object -> returns true
-        assertTrue(findFirstCommand.equals(findFirstCommand));
+        assertEquals(findFirstCommand, findFirstCommand);
 
         // same values -> returns true
-        FindCommand findFirstCommandCopy = new FindCommand(firstPredicate);
-        assertTrue(findFirstCommand.equals(findFirstCommandCopy));
+        FindCommand findFirstCommandCopy = new FindCommand(firstKeywordPredicate,
+                                                            firstDatePredicate,
+                                                            firstRatingPredicate);
+        assertEquals(findFirstCommand, findFirstCommandCopy);
 
         // different types -> returns false
-        assertFalse(findFirstCommand.equals(1));
+        assertNotEquals(1, findFirstCommand);
```

Comment:
If the code coverage decrease is from this pull request, might this be the reason?
CodeCov might not realise that `FindCommand::equals` is being tested here implicitly.